src/group_theory/group_action/defs.lean

```diff
@@ -150,6 +150,48 @@ is_scalar_tower.smul_assoc x y z
 
 instance semigroup.is_scalar_tower [semigroup α] : is_scalar_tower α α α := ⟨mul_assoc⟩
 
+namespace has_scalar
+variables [has_scalar M α]
+
+/-- Auxiliary definition for `has_scalar.comp`, `mul_action.comp_hom`,
+`distrib_mul_action.comp_hom`, `module.comp_hom`, etc. -/
+@[simp, to_additive has_vadd.comp.vadd
+/-" Auxiliary definition for `has_vadd.comp`, `add_action.comp_hom`, etc. "-/]
```

Comment:
```suggestion
/--" Auxiliary definition for `has_vadd.comp`, `add_action.comp_hom`, etc. "-/]
```

Reply:
No, that's not the problem. `/-" ... "-/` is just notation for a string, and `to_additive` uses that string for the docstring. I might have messed up something in how `to_additive` handles `has_scalar -> has_vadd`

Reply:
Ah, thanks for clearing up my misconception

Reply:
Maybe my change to set the name explicitly will help